File: Python3/프로그래머스/1/42576. 완주하지 못한 선수/완주하지 못한 선수.py
def solution(participant, completion):
    
    hashdic={}
    hashSum=0
    for par in participant:
        hashdic[hash(par)]=par
        hashSum+=hash(par)
    for com in completion:
        hashSum-=hash(com)
        
    return hashdic[hashSum]

File: JavaScript/프로그래머스/1/42576. 완주하지 못한 선수/완주하지 못한 선수.js
function solution(participant, completion) {
    const map = new Map();
    for(let x of participant){
        map.set(x, (map.get(x) || 0) + 1);
    }
    for(let x of completion){
        map.set(x, (map.get(x) || 0) - 1);
    }
    for(let[k,v] of map){
        if(v==1){
            return k;
        }
    }
   
}